src/web/components/SubNav/SubNav.tsx

```diff
@@ -1,87 +1,207 @@
 
-        const { showMore, isExpanded } = this.state;
-        const collapseWrapper = !showMore || !isExpanded;
-        const expandSubNav = !showMore || isExpanded;
-
-        return (
-            <div>
-                <Inner
-                    links={subnav.links}
-                    pillar={pillar}
-                    parent={subnav.parent}
-                    showMore={showMore}
-                    isExpanded={isExpanded}
-                    collapseWrapper={collapseWrapper}
-                    expandSubNav={expandSubNav}
-                    ulRef={this.ulRef}
-                    toggle={this.toggle}
-                    currentNavLink={currentNavLink}
-                />
-                <GuardianLines pillar={pillar} />
+    return (
+        <div>
+            <div className={cx({ [wrapperCollapsed]: collapseWrapper })}>
+                <ul
+                    ref={ulRef}
+                    className={cx({
+                        [subnavCollapsed]: !expandSubNav,
+                        [subnavExpanded]: expandSubNav,
+                    })}
+                >
+                    {subNavSections && subNavSections.parent && (
+                        <li
+                            key={subNavSections.parent.url}
+                            className={cx(ps1, psp[pillar])}
+                        >
+                            <a
+                                className={parentLinkStyle}
+                                href={subNavSections.parent.url}
+                            >
+                                {subNavSections.parent.title}
+                            </a>
+                        </li>
+                    )}
+                    {subNavSections &&
+                        subNavSections.links.map(link => (
+                            <li key={link.url}>
+                                <a
+                                    className={cx(linkStyle, {
+                                        [selected]:
+                                            link.title === currentNavLink,
+                                    })}
+                                    href={link.url}
+                                    data-link-name={`nav2 : subnav : ${trimLeadingSlash(
+                                        link.url,
+                                    )}`}
+                                >
+                                    {link.title}
+                                </a>
+                            </li>
+                        ))}
+                </ul>
+                {showMore && (
+                    <button
+                        onClick={() => setIsExpanded(!isExpanded)}
+                        className={moreStyle}
+                    >
+                        {isExpanded ? 'Less' : 'More'}
+                    </button>
+                )}
             </div>
-        );
-    }
-}
+            <GuardianLines pillar={pillar} />
```

Comment:
Hello, what's this doing here? This component appears to be lost and should be gently led to the layouts file

Reply:
good point!!